src/less/progress-spinner/base/progress-spinner.less

```diff
@@ -1,6 +1,11 @@
+@import "../../mixins/utility/utility-mixins.less";
+
+.progress-spinner,
 .spinner {
+    .customBackgroundImageProperty(progress-spinner-image-url);
+    //TODO remove spinner
     animation: spin 600ms linear infinite;
-    background-image: url(@spinner-image-url);
+    // background-image: var(--progress-spinner-image-url);
```

Comment:
You can remove this commented code now. Same for line 31.

Reply:
done, removed comments